From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Fri, 4 Feb 2022 17:07:02 -0500
Subject: [PATCH] Don't override GR_PYTHON_DIR.

---
 python/CMakeLists.txt | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index 5c142ed..8d5a578 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -14,16 +14,6 @@ if(NOT PYTHONINTERP_FOUND)
     return()
 endif()
 
-message(STATUS "CMAKE_INSTALL_PREFIX: ${CMAKE_INSTALL_PREFIX}")
-message(STATUS "GR_PYTHON_DIR: ${GR_PYTHON_DIR}")
-
-execute_process(
-    COMMAND ${PYTHON_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/get_python_lib.py" "${CMAKE_INSTALL_PREFIX}"
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-    OUTPUT_VARIABLE GR_PYTHON_DIR
-)
-message(STATUS "GR_PYTHON_DIR: ${GR_PYTHON_DIR}")
-
 add_subdirectory(bindings)
 
 ########################################################################
-- 
2.34.1

